examples/oraclelinux.yaml

```diff
@@ -0,0 +1,17 @@
+# Licence: https://www.oracle.com/downloads/licenses/oracle-linux-license.html
+arch: "x86_64"
+
+images:
+- location: "https://yum.oracle.com/templates/OracleLinux/OL8/u5/x86_64/OL8U5_x86_64-olvm-b113.qcow2"
```

Comment:
Could you add the link to https://yum.oracle.com/oracle-linux-templates.html as a comment line?